src/components/cylc/tree/TreeItem.vue

```diff
@@ -18,29 +18,26 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 <template>
   <div
     v-show="!filteredOutNodesCache.get(node)"
-    class="treeitem"
+    class="c-treeitem"
   >
     <div
       class="node d-flex align-center"
       :class="nodeClass"
       :style="nodeStyle"
     >
       <!-- the node's left icon; used for expand/collapse -->
-      <v-btn
+      <svg
         v-if="renderExpandCollapseBtn"
         aria-label="Expand/collapse"
-        aria-hidden="false"
         class="node-expand-collapse-button flex-shrink-0"
```

Comment:
This automatically adds the rotation transition back in, which I feel is a useful bit of visual feedback
```suggestion
        class="node-expand-collapse-button flex-shrink-0 v-btn"
```

Reply:
Not sure it's a good idea to apply v-btn styling to something which isn't even a `<button />`.

I'm not overly keen on the animation anyway, but would be happy to add this with CSS if desired.